daemon/cmd/devices_test.go

```diff
@@ -7,7 +7,6 @@
 package cmd
 
 import (
-	"fmt"
```

Comment:
Nit: this should be moved to the previous commit

Reply:
Done.